doc/doxygen/install/install-linux.doxygen

```diff
@@ -203,6 +206,14 @@
   mkdir OpenMS-build
   cd OpenMS-build
   \endcode
+  
+  Install the qt4-webkit, qt4-dev-tools and libqtwebkit-dev
+
+   \code
+   sudo apt-get install qt4-webkit
+   sudo apt-get qt4-deb-tools
+   sudo apt-get libqtwebkit-dev
+   \endcode
```

Comment:
`qt4-default` and `qt4-deb-tools` (typo there BTW) do not seem to make a big difference, except that the former uses a later compiler?
So the only difference is `qt4-webkit` (which I could not find on Debian by exact name). Did you mean `libqt4-webkit`? So, `libqtwebkit-dev` is not enough?